src/views/ExplorePage.vue

```diff
@@ -287,6 +304,25 @@ export default {
           }
         });
       }
+      Http.request({
+        method: "GET",
+        url: "https://scratchstats.com/api2/scratchdb/topusers/followers?country=global&page=0",
+        headers: {
+          Referer: "https://scratchstats.com/",
+          "User-Agent":
```

Comment:
Why is the UA modified?

Reply:
Honestly not sure - probably was trying to fix some network bug.

Reply:
Well, please remove this.